apps/input-descriptor-to-credential/src/modules/converter/dtos/input-descriptor-to-credential-response.dto.ts

```diff
@@ -18,10 +18,11 @@
 import { CredentialDto } from './credential.dto';
 import { Type } from 'class-transformer';
 import { ValidateNested } from 'class-validator';
+import { ApiProperty } from '@nestjs/swagger';
 
 export class InputDescriptorToCredentialResponseDto {
   @Type(() => CredentialDto)
-  @ValidateNested()
```

Comment:
Do we not want `@ValidateNested` anymore?

Reply:
This is response DTO, so the `class-validator` is not executed on this.